prof/src/com/oracle/souffleprof/ProgramRun.java

```diff
@@ -521,25 +521,32 @@ public Relation getRelation(String name) {
         Object[][] new_table = new Object[table.length][table[0].length];
         for (int i = 0; i < table.length; i++) {
             for (int j = 0; j < table[0].length; j++) {
-                if (table[i][j] == null) continue;
                 if (table[i][j] instanceof Double) {
                     new_table[i][j] = formatTime((Double)table[i][j]);
                 } else if (table[i][j] instanceof Long) {
                     new_table[i][j] = formatNum(precision, (Long)table[i][j]);
                 } else {
                     new_table[i][j] = table[i][j];
                 }
+                if (new_table[i][j] == null) {
+                    new_table[i][j] = "-";
+                }
             }
         }
         return new_table;
     }
 
     public String formatNum(int precision, Object number) {
-
+        Long amount;
+        if (number != null && number instanceof Long) {
+            amount = (Long) number;
+        } else {
+            return "0";
+        }
         if (precision == -1) {
-            return ""+number;
+            return ""+amount;
```

Comment:
Would probably prefer Long.toString(amount) rather than the implicit concat conversion